Description: Disables the sampling profiler in WebKit (fails to build on arm64)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/modules/javafx.web/src/main/native/Source/cmake/WebKitFeatures.cmake
+++ b/modules/javafx.web/src/main/native/Source/cmake/WebKitFeatures.cmake
@@ -67,13 +67,13 @@
         set(ENABLE_FTL_DEFAULT ON)
         set(USE_SYSTEM_MALLOC_DEFAULT OFF)
         set(ENABLE_C_LOOP_DEFAULT OFF)
-        set(ENABLE_SAMPLING_PROFILER_DEFAULT ON)
+        set(ENABLE_SAMPLING_PROFILER_DEFAULT OFF)
     elseif (WTF_CPU_ARM AND WTF_OS_LINUX AND ARM_THUMB2_DETECTED)
         set(ENABLE_JIT_DEFAULT ON)
         set(ENABLE_FTL_DEFAULT OFF)
         set(USE_SYSTEM_MALLOC_DEFAULT OFF)
         set(ENABLE_C_LOOP_DEFAULT OFF)
-        set(ENABLE_SAMPLING_PROFILER_DEFAULT ON)
+        set(ENABLE_SAMPLING_PROFILER_DEFAULT OFF)
     elseif (WTF_CPU_MIPS AND WTF_OS_LINUX)
         set(ENABLE_JIT_DEFAULT ON)
         set(ENABLE_FTL_DEFAULT OFF)
